keras_nlp/models/gpt_neox/rotary_embedding.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2023 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import tensorflow as tf
+from tensorflow import keras
+
+# referenc from https://github.com/huggingface/transformers/blob/17a55534f5e5df10ac4804d4270bf6b8cc24998d/src/transformers/models/esm/modeling_tf_esm.py#L93
+
+
+class RotaryEmbedding(keras.layers.Layer):
+    def __init__(self, hidden_dim):
+        super().__init__()
+        self.hidden_dim = hidden_dim
+
+    def build(self, input_shape):
+        super().build(input_shape)
+        self.inverse_freq = self.add_weight(
+            "inverse_freq", shape=(self.hidden_dim // 2,), dtype=tf.float32
+        )
+
+        self.inverse_freq.assign(
```

Comment:
just drive by nit, but if you break up this line into a few lines it will probably format way better with black. e.g.

```
range = tf.range(start=0, limit=self.hidden_dim, delta=2, dtype="float32")
self.inverse_freq.assign(1.0 / ...)
```